CDCgov/prime-reportstream
# Remove extra <hr>s wrapping breadcrumb component, fixes #8252

This PR ...

Removes extra `<hr>`s wrapping breadcrumb component. 

Test Steps:
1. Go to any page under /resources/* or /support/*
1. See extra `<hr>`s have been removed from around breadcrumb/`<h1>`

## Changes
- Removes two extra `<hr>`s in `frontend-react/src/components/Crumbs.tsx`

![Screen Shot 2023-02-07 at 9 29 37 PM](https://user-images.githubusercontent.com/57154955/217590070-36fb0ef7-86cb-4097-897a-481a92a50641.png)

![Screen Shot 2023-02-08 at 11 23 00 AM](https://user-images.githubusercontent.com/57154955/217589672-48107941-de4c-441c-9703-634d20b9f51b.png)


## Checklist

### Testing

### Process

## Linked Issues
- Fixes #8252 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
